scipy/stats/_continuous_distns.py

```diff
@@ -6384,6 +6384,37 @@ def _munp(self, n, c):
     def _entropy(self, c):
         return np.log(2.0/c) + 1.0
 
+    @_call_super_mom
+    def fit(self, data, *args, **kwds):
+        data, fc, floc, fscale = _check_fit_input_parameters(self, data,
+                                                             args, kwds)
+
+        # Specialize MLE only when location is known.
+        if floc is None:
+            return super(type(self), self).fit(data, *args, **kwds)
+
+        # Raise an error if any observation has zero likelihood.
+        if np.any(data <= floc):
+            raise FitDataError("loglaplace", lower=floc, upper=np.inf)
+
+        # Remove location from data.
+        if floc != 0:
+            data = data - floc
+
+        # When location is zero, the log-Laplace distribution is related to
+        # the Laplace distribution in that if X ~ Laplace(loc=a, scale=b),
+        # then Y = exp(X) ~ LogLaplace(c=1/b, loc=0, scale=exp(a)).  It can
+        # be shown that the MLE for Y is the same as the MLE for X = ln(Y).
```

Comment:
I think this is true in general for log transformed distributions due to invariance properties of MLE, right?

What do you think of calling `laplace.fit` instead of repeating the formulas?

If they were any more complicated I think it would be more important, but I suppose if we do we'd have to transform parameters back and forth. Up to you.

Reply:
Yes it’s in general true for monotone transform of the random variable where the transform itself doesn’t involve unknown parameters.

It’s probably better to call `laplace.fit` instead, in case the latter gains more features in the future. It also helps to reduce the surface for testing.